nexp/clients/data.py

```diff
@@ -353,12 +353,20 @@ def candidates_for_facility(self, facility: Any) -> ModelIterator:
                   AND json_extract(c.fields, "$.unavailable") is null
                       {clause}
 
+            ), tracked_candidates AS (
+                SELECT substring_index(tracked_candidate.value, ',', 1) AS name
```

Comment:
this is my attempt to extract out the candidate's full name before their phone number since each value is formatted as follows:

`FirstName Surname, (555)-444-111`

Reply:
These values should come back as record ids because it's a linked field (I think?) so you probably won't need to do sub-stringing here.